src/t8_schemes/t8_default/t8_default_vertex/t8_default_vertex.cxx

```diff
@@ -313,8 +313,8 @@ int
 t8_default_scheme_vertex::element_is_valid (const t8_element_t *elem)
 
 {
-  const t8_dvertex *v = (const t8_dvertex_t *) elem;
-  return 0 <= v->level && v->level <= T8_DVERTEX_MAXLEVEL;
+  return 1;
+  //return 0<=level && level <= maxlevel
```

Comment:
?

Reply:
That is actually a very cool thing, that @lukasdreyer found with this clean-up. A vertex only stores a level, which is an 8-bit unsigned int. The maximal level of a vertex is 255, which is the maximum of an unsigned int, so a vertex is per definition of its datastructure always valid.